src/c_kzg_4844.c

```diff
@@ -990,14 +990,25 @@ C_KZG_RET blob_to_kzg_commitment(
     return C_KZG_OK;
 }
 
+/**
+ * Struct holding the argument to verify_kzg_proof_impl.
+ * Having a struct holding these allows some code simpliciations
```

Comment:
Minor nit, punctuation & typo.
```suggestion
 * Having a struct holding these allows some code simplifications.
```